dvc/_debug.py

```diff
@@ -56,14 +56,17 @@ def debug():
 
 
 @contextmanager
-def debugtools(args):
+def debugtools(args: "argparse.Namespace" = None, **kwargs):
+    kw = vars(args) if args else {}
+    kw.update(kwargs)
+
     with ExitStack() as stack:
-        if args.pdb:
+        if kw.get("pdb"):
             stack.enter_context(debug())
-        if args.cprofile:
-            stack.enter_context(profile(args.cprofile_dump))
-        if args.instrument:
-            stack.enter_context(instrument(args.instrument_open))
+        if kw.get("cprofile") or kw.get("cprofile_dump"):
+            stack.enter_context(profile(kw.get("cprofile_dump")))
+        if kw.get("instrument") or kw.get("instrument_open"):
+            stack.enter_context(instrument(kw.get("instrument_open", False)))
```

Comment:
Both --cprofile` and `--cprofile-dump` flags are not required from now on. Similar behaviour for `--instrument(-*)`